lib/init/config-initializer.js

```diff
@@ -50,6 +51,16 @@ function writeFile(config, format) {
         extname = ".json";
     }
 
+    const pkgJSONPath = npmUtils.findPackageJson();
+
+    if (pkgJSONPath) {
+        const pkgJSONContents = JSON.parse(fs.readFileSync(pkgJSONPath, "utf8"));
```

Comment:
why not just `require(pkgJSONPath)`?

Reply:
`require(pkgJSONPath)` will cache the file.

Reply:
well, I see. thx!